README.md

```diff
@@ -55,6 +55,25 @@ functions:
         period: 60
         evaluationPeriods: 1
 ```
+## Metric Log Filters
+You can monitor a log group for a function for a specific pattern. Do this by adding the pattern key.
+You can learn about custom patterns at: http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html 
+
+The following would create a custom metric log filter based alarm named "barAlarm". Any function that included this alarm would have its logs scaned for the pattern 'exception Bar' and if found would trigger an alarm.
+
+```yaml
+    custom:
+        alerts:
```

Comment:
Since the alert is tied to the function, maybe we could attach this to the functions key?

```yaml

functions:
  foo:
    alerts:
      {same as below}
```